mempool/cache_bench_test.go

```diff
@@ -17,7 +19,7 @@ func BenchmarkCacheInsertTime(b *testing.B) {
 	b.ResetTimer()
 
 	for i := 0; i < b.N; i++ {
-		cache.Push(txs[i])
+		cache.Push(types.Tx(txs[i]).Key())
```

Comment:
This made sense to me at first, but wasn't this included in the old benchmark? also, moot now, as the change to use key instead of just the transaction and then calculate the key inside `Push` was reverted in https://github.com/celestiaorg/celestia-core/pull/893/commits/75a5564f7374cc8dcf4a70fedad045eebd5ae5ec